README.md

```diff
@@ -37,3 +37,21 @@ BaconQrCode comes with multiple back ends for rendering images. Currently includ
 - `ImagickImageBackEnd`: renders raster images using the Imagick library
 - `SvgImageBackEnd`: renders SVG files using XMLWriter
 - `EpsImageBackEnd`: renders EPS files
+
+### GDLib Renderer
+GD library has so many limitations, that GD support is not added as backend, but as separated rendered.
+Use `GDLibRendered` instead of `ImageRenderer`. Here are limitations:
```

Comment:
`s/Rendered/Renderer/`

Reply:
Oh, and: "**These** are **the** limitations" :)

Reply:
Fixed